docs-content/release-notes.html.md.erb

```diff
@@ -3,10 +3,16 @@ title: Release Notes for VMware Tanzu Observability by Wavefront Nozzle
 owner: Partners
 ---
 
-<strong>Page last updated: November 18, 2021</strong>
+<strong>Page last updated: December 10, 2021</strong>
 
 These are release notes for VMware Tanzu Observability by Wavefront Nozzle.
 
+##<a id="3-0-3"></a> v3.0.3
+
+**Release Date:** December 10, 2021
+
+* Security fix for a vulnerability
```

Comment:
Should be 

* Security fix for a vulnerability.

Reply:
Text has been updated.